facade/service.go

```diff
@@ -1920,13 +1936,14 @@ func (f *Facade) restoreIPs(ctx datastore.Context, svc *service.Service) error {
 func (f *Facade) AssignIPs(ctx datastore.Context, request addressassignment.AssignmentRequest) error {
 	defer ctx.Metrics().Stop(ctx.Metrics().Start("Facade.AssignIPs"))
 	visitor := func(svc *service.Service) error {
+		alog := f.auditLogger.Action(audit.Update).Message(ctx, "Assigning IP for Service: " + svc.Name).ID(svc.GetID()).Type(svc.GetType())
```

Comment:
You changed `domain.service` to implement the `Entity` interface, so you can set the type and id like so:
```
alog := f.auditLogger.Action(audit.Update).
    Message(ctx, "Assigning IP for Service: " + svc.Name).Entity(svc)
```